index.js

```diff
@@ -52,6 +52,16 @@ function serveSinglePageApp(request) {
   request = stripQueryString(request)
   request = mapRequestToAsset(request)
 
+  const url = new URL(request.url);
```

Comment:
@eramus @PorlyBe moved the logic here; does this look ok?

Reply:
yeah i think that looks good